src/haddock/libs/libalign.py

```diff
@@ -228,9 +228,12 @@ def load_coords(pdb_f, atoms, filter_resdic=None, numbering_dic=None):
                     idx += 1
     chain_ranges = {}
     for chain in chain_dic:
-        min_idx = min(chain_dic[chain])
-        max_idx = max(chain_dic[chain])
-        chain_ranges[chain] = (min_idx, max_idx)
+        if chain_dic[chain] == []:
+            raise ALIGNError(f"Chain matching error on {pdb_f}, chain {chain}")
```

Comment:
I suggest adding a very brief comment stating when such a situation happens. When PDB have no chain ID? Equal chain ID? Something of this kind.